app/__init__.py

```diff
@@ -165,6 +166,13 @@ def get_locations_of_events():
 
     return dict(locations=get_locations_of_events)
 
+
+@app.context_processor
+def get_key_settings():
```

Comment:
Just noticed, The settings are already being loaded into a `settings` variable ... (you can find that by scrolling down ... around line `200` ) ... You can use that variable